lib/utils/extensions/geopoint_extensions.dart

```diff
@@ -0,0 +1,8 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:google_maps_flutter/google_maps_flutter.dart";
+
+extension ToLatLng on GeoPoint {
```

Comment:
Very smart!
I needed this at some point :)
Maybe we can add a task  in the backlog, to use it everywhere.

Reply:
I don't think that's necessary, we can just hope that people see it and use it in the future.